packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -164,7 +164,7 @@ export const Query = {
 
     const after = maybeAfter ? Number(maybeAfter) : 0
     const searchArgs: Omit<SearchArgs, 'type'> = {
-      page: Math.ceil(after / first),
+      page: Math.ceil(after / first) || 0,
```

Comment:
should we also check after and first values?
first shouldn't be 0.

Reply:
we double check the first in the search already (the other file of this PR)